### ISSUE TITLE
[BUG] Battery and time up against each other when using AOSP Mods.

### ISSUE THREAD

**Issue by devanteweary** [Mahmud0808/Iconify#112]
### Describe the bug

Hello,

I have a bug that has something to do with the behavior between Iconify and AOSP Mods.
I have a small visual issue with the status bar battery and clock.

The battery and clock icons are right up against each other.
I don't even think there's a pixel between their "boxes".
I've tried different battery styles and turning percentage off and on.

I know this one isn't necessarily Iconify nor AOSP but rather something going on between the two.

Just wanted to post then in hopes someone else had the same issue and fixed it!

Thank you!

### Steps to reproduce

If I disable Iconify, the spacing between the two returns to normal.

In Iconify...
I've tried changing any settings that had to do with the status bar.
I also made various changes to colors just as a test.

In AOSP Mods...
I've tried resizing both the clock and the battery.
As for the battery, when it resizes, it does so from the right. Like it's anchored to the right.
I've tried custom text. 
I've also tried messing with the status bar padding.
I've tried pretty much every option I could see that had to do with the status bar.

### Expected behavior

Normal spacing.

### Screenshots

![Screenshot_20230309-102231](https://user-images.githubusercontent.com/29805497/228059147-c0f79998-bb03-44e6-b7d8-d1f489a3ae18.png)

### Device information

Versions
- Pixel 6 Pro
- Android 12 - July 5, 2022 security update
- Magisk v25.2 Stable
- LSPosed v1.8.6 (6712) - Zygisk
- AOSP-Full v2.4.1 

Other Magisk modules I have installed:
- Iconify v6.1.0
- MagiskHide v1.10.3
- mindetach v3.3
- Universal SafetyNet Fix v2.4.0-MOD_1.2
- Zygisk - LSPosed v1.8.6 (6712)

**Comment by Mahmud0808** [comment]
> If I disable Iconify, the spacing between the two returns to normal.

Did you mean disabling iconify module from LSPosed or the magisk module?

What happens if you disable AOSPMods and only use Iconify? Does the issue still persist?

**Comment by devanteweary** [comment]
> > If I disable Iconify, the spacing between the two returns to normal.
> 
> Did you mean disabling iconify module from LSPosed or the magisk module?
> 
> What happens if you disable AOSPMods and only use Iconify? Does the issue still persist?

I disabled it in Magisk. It did disable everything related to Iconify on my phone though.

If I disable AOSPMods, I can't really say because the clock goes back to the left.
AOSPMods is what lets you put the clock on the right.

But I did find a work around that has nothing to do with Iconify so everything's cool now. 
I used AOSPMods to put special blank space characters as "custom text" next to the clock so it simulates the padding.

Thank you for responding!
